zerver/data_import/rocketchat.py

```diff
@@ -37,12 +38,10 @@
 from zerver.models import Reaction, RealmEmoji, Recipient, UserProfile
 
 
-def make_realm(
-    realm_id: int, realm_subdomain: str, domain_name: str, rc_instance: dict[str, Any]
-) -> ZerverFieldsT:
-    created_at = float(rc_instance["_createdAt"].timestamp())
+def make_realm(realm_id: int, realm_subdomain: str, domain_name: str) -> ZerverFieldsT:
+    NOW = float(timezone_now().timestamp())
```

Comment:
This change will work, but I'd like to confirm why `rc_instance["_createdAt"]` is no longer available. Did rocketchat moved this data somewhere else. I'll do this once I arrange a fresh database dump (see my check-in)

Reply:
See [CZO message](https://chat.zulip.org/#narrow/stream/9-issues/topic/Migration.20from.20Rocket.2EChat.20issues.3A.20messages.20and.20uploads/near/1919196)